pymchelper/executor/options.py

```diff
@@ -43,13 +51,24 @@ class SimulationSettings:
     (i.e. whether this is SHIELD-HIT12A input or FLUKA input)
     """
 
-    def __init__(self, input_path, simulator_exec_path=None, cmdline_opts=None):
+    def __init__(self, input_path, simulator_type=SimulatorType.shieldhit, simulator_exec_path=None, cmdline_opts=None):
+        # simulator type (shieldhit, topas or fluka)
+        self.simulator_type = simulator_type
```

Comment:
```suggestion
        self.simulator_type : SimulatorType = simulator_type
```

is that correct typehint ?

Reply:
yes